hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueueService.kt

```diff
@@ -71,7 +72,7 @@ class HmppsQueueService(
     findByQueueName(queueName)
       ?.let { hmppsQueue -> PurgeQueueRequest(hmppsQueue.queueName, hmppsQueue.sqsClient, hmppsQueue.queueUrl) }
       ?: findByDlqName(queueName)
-        ?.let { hmppsQueue -> PurgeQueueRequest(hmppsQueue.dlqName, hmppsQueue.sqsDlqClient, hmppsQueue.dlqUrl) }
+        ?.let { hmppsQueue -> PurgeQueueRequest(hmppsQueue.dlqName!!, hmppsQueue.sqsDlqClient!!, hmppsQueue.dlqUrl!!) }
```

Comment:
Pants.  I think what you've done is right but these bangbangs are happening all over the place - when in reality either all dlq attributes are null or none are.  I think we might need to change the architecture slightly... but that's for another day.

Reply:
I know - It's not great.